### ISSUE TITLE
Load the model Error

### ISSUE THREAD

**Issue by gyzz** [da03/Attention-OCR#22]
Hi, I have run the code you supported. When I test the pre-trained model, the command is:
`python src/launcher.py --phase=test --visualize --data-path=evaluation_data/svt/test.txt --data-base-dir=evaluation_data/svt --log-path=log.txt --load-model --model-dir=model --output-dir=results --old-model-version`
But the result shows:
```
NotFoundError (see above for traceback): Tensor name "bidirectional_rnn/bw/basic_lstm_cell/biases" not found in checkpoint files model/translate.ckpt-47200
         [[Node: save/RestoreV2_25 = RestoreV2[dtypes=[DT_FLOAT], _device="/job:localhost/replica:0/task:0/cpu:0"](_recv_save/Const_0, save/RestoreV2_25/tensor_names, save/RestoreV2_25/shape_and_slices)]]
         [[Node: save/RestoreV2_33/_99 = _Recv[client_terminated=false, recv_device="/job:localhost/replica:0/task:0/gpu:0", send_device="/job:localhost/replica:0/task:0/cpu:0", send_device_incarnation=1, tensor_name="edge_272_save/RestoreV2_33", tensor_type=DT_FLOAT, _device="/job:localhost/replica:0/task:0/gpu:0"]()]]
```
when I test the model that I trained on the dataset you supported, the command is:
`python src/launcher.py --phase=test --visualize --data-path=evaluation_data/svt/test.txt --data-base-dir=evaluation_data/svt --log-path=log.txt --load-model --model-dir=train --output-dir=results`
But there is another error:
```
DataLossError (see above for traceback): Unable to open table file train_1/translate.ckpt-71000.data-00000-of-00001: Data loss: not an sstable (bad magic number): perhaps your file is in a different file format and you need to use a different restore operator?
         [[Node: save/RestoreV2_4 = RestoreV2[dtypes=[DT_FLOAT], _device="/job:localhost/replica:0/task:0/cpu:0"](_recv_save/Const_0, save/RestoreV2_4/tensor_names, save/RestoreV2_4/shape_and_slices)]]
         [[Node: save/RestoreV2_33/_99 = _Recv[client_terminated=false, recv_device="/job:localhost/replica:0/task:0/gpu:0", send_device="/job:localhost/replica:0/task:0/cpu:0", send_device_incarnation=1, tensor_name="edge_272_save/RestoreV2_33", tensor_type=DT_FLOAT, _device="/job:localhost/replica:0/task:0/gpu:0"]()]]
```
I found both the error occur in the line of src/model/model.py: ` self.saver_all.restore(self.sess, ckpt.model_checkpoint_path)`. Do you know what's the problem is? Thank you.

**Comment by da03** [comment]
@gyzz Did you figure out where the problem is? I would sugget you updating tensorflow and keras to the latest version.

**Comment by gyzz** [comment]
@da03 Thank you for  your reply. Now I can test the model after updating the cuda to the version8. I use the tensorflow0.11 and Keras1.1.1 all the time. But the latest version is tensorflow0.12 and Keras1.1.2. Do you mean the code also supports the latest version? By the way, I have another problem. I can test your pre-trained model normally and the accuracy is 30.91% (200 out of 647 correct). But when I test the model that I trained by myself, the accuracy is 0, including testing on the training dataset. I think there are some errors in the training. The command that I use is
`python src/launcher.py --phase=train --data-path=sample/sample.txt --data-base-dir=sample --log-path=log_train.txt --no-load-model`
And the model has already converged:
```
2016-12-13 19:35:51,224 root  INFO     current_step: 70994
2016-12-13 19:35:51,470 root  INFO     step_time: 0.245469, step_loss: 0.000006, step perplexity: 1.000006
2016-12-13 19:35:51,550 root  INFO     current_step: 70995
2016-12-13 19:35:51,735 root  INFO     step_time: 0.185029, step_loss: 0.000031, step perplexity: 1.000031
2016-12-13 19:35:51,754 root  INFO     current_step: 70996
2016-12-13 19:35:52,007 root  INFO     step_time: 0.253041, step_loss: 0.000008, step perplexity: 1.000008
2016-12-13 19:35:52,104 root  INFO     current_step: 70997
2016-12-13 19:35:52,366 root  INFO     step_time: 0.261504, step_loss: 0.000010, step perplexity: 1.000010
2016-12-13 19:35:52,408 root  INFO     current_step: 70998
2016-12-13 19:35:52,587 root  INFO     step_time: 0.178716, step_loss: 0.000013, step perplexity: 1.000013
2016-12-13 19:35:52,738 root  INFO     current_step: 70999
2016-12-13 19:35:52,899 root  INFO     step_time: 0.161081, step_loss: 0.000013, step perplexity: 1.000013
2016-12-13 19:35:52,900 root  INFO     global step 71000 step-time 0.22 loss 0.000023  perplexity 1.00
2016-12-13 19:35:52,900 root  INFO     Saving model, current_step: 71000
```
Do you think it is also the problem of version?
